src/api/ResourcefulApi.js

```diff
@@ -63,16 +63,14 @@ export class ResourcefulApi extends Api {
 
   /**
    *
-   * @param {Array} modulesToRegister
+   * @param {Array} apiModulesToRegister
    */
-  setupModules (modulesToRegister = []) {
-    console.time('api: setup modules')
+  setupApiModules (apiModulesToRegister = []) {
+    console.time('api: setup api modules')
 
-    for (const moduleName in modulesToRegister) {
-      this.registerModule(moduleName, this[moduleName])
-    }
+    apiModulesToRegister.forEach(name => this.registerModule(moduleName, this[moduleName]))
```

Comment:
Should this be preceded by a type check maybe? Might be useful to inform devs that they have to pass an array.

Reply:
I'd say that the default parameter is sufficient and serves as a type hint for devs.